docs/overview.md

```diff
@@ -0,0 +1,98 @@
+
+Learn more about [packaging your applications](/docs/packaging-applications/index.md) or how to automate your existing tool chain to push packages to your Octopus Deploy server with our [Build Server Integrations](/docs/octopus-rest-api/index.md).
+
+## The Deployment Process
+
+Octopus Deploy is designed to work with teams following agile software development methodologies, that is, continuously deploying software, iterating, making changes, and redeploying.
+
+Before you can deploy your software, you need to create a project which will include your deployment process and all the information needed for your teams to successfully redeploy every time. The deployment process is a series of steps, each step contains a specific action (or set of actions) that is executed as part of the deployment process every time your software is deployed. Octopus Deploy includes a range of built in steps you can use in your deployment processes.
+
+As part of the deployment process, you can define variables that change based on the scope of your deployments, this means, for instance, you can use different connection strings when you deploy to different environments.
+
+After the initial setup, your deployment process shouldn't change between deployments even though the software being deployed will change as part of the development process.
+
+Learn more about the [deployment process](/docs/deployment-process/index.md), how you can set up [projects](/docs/deployment-process/projects/index.md), use [variables](/docs/deployment-process/variables/index.md) and [deploy releases](/docs/deployment-process/releases/index.md).
+
+## Deploy your Software
+
+Once you have access to an Octopus Server, your infrastructure is configured, your applications packaged, and the deployment process defined, you're ready to start deploying your software. To deploy your software, you first create a release.
+
+A release is a snapshot of your deployment process, configuration variables, and software packages. Releases are created from projects and deployed to your environments.
+
+When you deploy a release, you are executing the deployment process with all the associated details, as they existed when the release was created. You can deploy a release as many times as you want to.
+
+Learn more about how [deploying releases](/docs/deployment-process/releases/index.md), or see some [deployment examples](/docs/deployment-examples/index.md).
+
+## The Delivery Pipeline
+
+A typical workflow using Octopus Deploy could something like the following:
+
+1. **Commit Code to Your Existing Source Control System.**
+
+   You might be using Git, Team Foundation Server, Subversion, or Mercurial. The choice is yours.
+
+1. **Your CI/Build Server Compiles the Code and Runs Unit Tests.**
+
+   You might be using TeamCity, Jenkins, Bamboo, Azure DevOps, Team Foundation Server (TFS), or CruiseControl.NET. Again, the choice is yours.
```

Comment:
I like listing all of the possibilities but someone might take that to mean that we only support those. Maybe change the ending to "(TFS), CruiseControl.NET, or your preferred CI server"?